_functions_jitsi.sh

```diff
@@ -205,19 +224,57 @@ do_start_jitsi() {
     -e "TZ=UTC" \
     --network "${DEPLOYMENT_JITSI_NETWORK_NAME}" \
     --restart unless-stopped \
-    --name ${DEPLOYMENT_JITSI_JVB_CONTAINER_NAME} jitsi/jvb:latest
+    --name ${DEPLOYMENT_JITSI_JVB_CONTAINER_NAME} jitsi/jvb:stable-5142
+  echo_info "${DEPLOYMENT_JITSI_JVB_CONTAINER_NAME} container started"
+
+  echo_info "Starting Jitsi Jibri container ${DEPLOYMENT_JITSI_JIBRI_CONTAINER_NAME} based on image jitsi/jibri:stable-5142"
+  # Ensure there is no container with the same name
+  delete_docker_container ${DEPLOYMENT_JITSI_JIBRI_CONTAINER_NAME}
+  cp -v ${ETC_DIR}/jitsi/finalize.sh ${DEPLOYMENT_DIR}/finalize.sh
+  chmod +x ${DEPLOYMENT_DIR}/finalize.sh
+  ${DOCKER_CMD} run \
+    -d \
+    -v ${DEPLOYMENT_JITSI_JIBRI_CONTAINER_NAME}_config:/config:Z  \
+    -v /dev/shm:/dev/shm:Z  \
+    -v ${DEPLOYMENT_DIR}/finalize.sh:/tmp/finalize.sh \
+    --cap-add SYS_ADMIN \
+    --cap-add NET_BIND_SERVICE \
+    --device /dev/snd \
```

Comment:
Could you briefly explain this part? I see you used /dev/snd and /dev/shm . Thanks

Reply:
This configuration is required by Jitsi to enable recording:
- we mount /dev/shm to use shred memory for the chrome driver embedded in the jibri container
- we use /dev/snd to use the sound card of the server hosting the jibri container

Reply:
For more details:
- https://github.com/jitsi/docker-jitsi-meet/blob/master/jibri.yml
- https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#jitsi-broadcasting-infrastructure-jibri-configuration

Reply:
OK, thanks for the explanation. So consider that we have multiple instances running with Jitsi on the same server, will there be any side effect?

Reply:
I configured 5 capture interfaces to allow 5 instance to record at the same time as indicated in the second link above:
# configure 5 capture/playback interfaces
echo "options snd-aloop enable=1,1,1,1,1 index=0,1,2,3,4" > /etc/modprobe.d/alsa-loopback.conf
But I haven't test other instances since i must create a specific dns entry for each one like we did for plf-jitsi.exoplatform.org to have a valid certificate.

Reply:
IMO It would be better to deploy two instances and look for any side effects. But it sounds good! Good job @oussafi